src/Authenticator/CookieAuthenticator.php

```diff
@@ -51,6 +52,7 @@ class CookieAuthenticator extends AbstractAuthenticator implements PersistenceIn
             'name' => 'CookieAuth',
         ],
         'passwordHasher' => 'Authentication.Default',
+        'useSalt' => true,
```

Comment:
Since this changes how the cookie token is created by default, the PR needs to target `1.next`.

Reply:
Yeah, will do. Gotta figure out how to rebase my branch properly.

Reply:
Rebasing will be easier if you first squash all the new commits in your feature branch into a single commit.

Reply:
But we'd want it in `2.x` too, don't we? There's no `2.next` branch (yet).

Reply:
Err sorry, I mixed up the numbers. We are on `2.x` series currently, not `1.x` :)

So this is fine since we don't have a `2.next`. We can just tag a new minor after this is merged.

Reply:
Still targeting master, I'll resolve this converstation.